web/src/locales/en.json

```diff
@@ -262,48 +264,56 @@
   "country-history": {
     "carbonintensity": {
       "hourly": "Hourly carbon intensity",
+      "hourly_72": "Hourly carbon intensity",
```

Comment:
Do you think we can do this in a way that don't duplicate the translations?

It's not a problem to do it like this but if there is a *easy* way for us to avoid it I would.